situp-plugins/apmtracesource/build.gradle

```diff
@@ -7,6 +7,7 @@ dependencies {
     compile project(':situp-api')
     compile project(':situp-plugins:common')
     compile 'commons-codec:commons-codec:1.15'
+    testCompile "io.grpc:grpc-testing:${versionMap.grpc}"
```

Comment:
Do we need this? If its only for the @Rule I think we can avoid this because we do not have direct dependency on grpc and this is introducing it. I would like to have dependency only on armeria.

Reply:
Removed dependency. It's a nice to have but I wasn't able to find a way to keep Armeria's version of gRPC in sync with whichever version of grpc-testing we were pulling in.